### PR TITLE
Fix #129 cases data source

### LINKED ISSUE #129: Add new source for cases data
Since the links to the cases data files behind the Arcgis dashboard break without warning from time to time, we are using this now as an opportunity to switch the source of the cases data to https://github.com/robert-koch-institut/SARS-CoV-2_Infektionen_in_Deutschland

There are a couple of DAGs this data gets used in, and the new source does not contain e.g. the Bundesland, Landkreis, Altersgruppe2 (irrelevant) columns, so this will take a bit of digging.

### PR DESCRIPTION
**Pull request by davhin**
Will eventually fix https://github.com/rocs-org/data-pipelines/issues/129

### CODE REVIEW

**Comment by davhin** [comment]
Converted this to a draft again bc (i) cases update ran fine tonight so we can take more time here, and (ii) realised the query to upload data will have to be changed since the caseID is no longer existent, so so we either flush the table before upload (my preference), or figure out a unique combination of columns

File: services/airflow/src/lib/dag_helpers/test_write_dataframe_to_postgres.py [hunk 1]
```diff
@@ -71,6 +72,40 @@ def test_insert_dataframe_to_postgres_does_not_overwrite(db_context: DBContext):
...
 
+def test_truncate_insert_dataframe_to_postgres_does_replace_existing_data(
+    db_context: DBContext,
+):
+    connect_to_db_and_insert_pandas_dataframe(
+        schema="coronacases", table="german_counties_more_info", data=DATA4
+    )
+    connect_to_db_and_truncate_insert_pandas_dataframe(
+        schema="coronacases", table="german_counties_more_info", data=DATA4
+    )
```
Comment (jakobkolb) [inline services/airflow/src/lib/dag_helpers/test_write_dataframe_to_postgres.py:83]: Write different data the second time and test that only data of second write persists

**Review by jakobkolb — CHANGES_REQUESTED** [review]
Just one little thing

**Comment by jakobkolb** [comment]
> Converted this to a draft again bc (i) cases update ran fine tonight so we can take more time here, and (ii) realised the query to upload data will have to be changed since the caseID is no longer existent, so so we either flush the table before upload (my preference), or figure out a unique combination of columns

Updating this to a more stable data source makes sense imho. On the caseID thing: If we flush the table, we probably don't need it. The clean way would be to remove it via a db migration, but we could also just leave it empty.

File: services/airflow/src/lib/dag_helpers/test_write_dataframe_to_postgres.py (hunk 1 above)
Comment (davhin) [reply services/airflow/src/lib/dag_helpers/test_write_dataframe_to_postgres.py]: done

**Comment by davhin** [comment]
This now
- removes the 'caseid' and 'agegroup2' columns (necessary since 'caseid' is no longer provided agegroup2 was useless anyways)
- pulls the data from the repo of MF4
- adds 'state', 'stateid', 'county' based of 'countyid' from a mapping .csv that is based on the AGS data from the old cases source with additional Berliner Bezirke as pseudo Landkreise
- truncates the table coronacases.german_counties_more_info at every run, then adds the freshly pulled copy

**Review by jakobkolb — APPROVED** [review]
I like this.

**Pull request merged**